internal/data/quality_metrics.go

```diff
@@ -339,16 +370,17 @@ func (s *Store) GetMRsMergedWithoutReview(weeks []string, teamMembers []int64) (
 	mrCountByWeeks := make(map[string]MrCountByWeek)
 
 	for rows.Next() {
-		var mrweek MrCountByWeek
+		var week string
+		var count int32
 
-		if err := rows.Scan(&mrweek.Count, &mrweek.Week); err != nil {
+		if err := rows.Scan(&count, &week); err != nil {
```

Comment:
Not clear what is the benefit of removing the struc in favor for replacing with per-field vars. Just curios about the motivation

Reply:
I will restore previous,with struct